src/docs/output-targets/custom-elements.md

```diff
@@ -68,6 +68,15 @@ This flag defaults to `false` when omitted from a Stencil configuration file.
 > Note: At this time, components created not using JSX may not be automatically
   defined. This is a known limitation of the API and users should be aware of
   it
+### externalRuntime
```

Comment:
Taking a step back, I took a look at the diff of using `externalRuntime` in a default Stencil project (`npm init stencil component`) this morning where I built the project twice with the following Stencil config output target settings:
```
    {
      type: 'dist-custom-elements',
      externalRuntime: false,
    },
    {
      type: 'dist-custom-elements-bundle',
      externalRuntime: false,
    },
```
```
    {
      type: 'dist-custom-elements',
      externalRuntime: true,
    },
    {
      type: 'dist-custom-elements-bundle',
      externalRuntime: true,
    },
```

It seems like this flag really only does anything for CEB. Is that right? Should we re-evaluate if this flag should exist for the CE output target?

Reply:
@rwaskiewicz I went and double-checked this and was seeing differences in `dist-custom-elements` with the flag on and off. Most notably is the bundling where, when `true`, the Stencil core code is not included in the bundle

Reply:
@rwaskiewicz I didn't run a diff, I was just looking at the output code when running with `externalRuntime: true` and `externalRuntime: false`. Were you not talking about the output? Maybe we can sync on this tomorrow morning?